snapcraft/internal/sources.py

```diff
@@ -383,6 +403,8 @@ def _get_source_type_from_uri(source, ignore_errors=False):
         source_type = 'tar'
     elif source.endswith('.zip'):
         source_type = 'zip'
+    elif source.startswith('svn:'):
+        source_type = 'subversion'
```

Comment:
another nit, please move this case after the git one. I'm a little code order freak, and I would like that all the cvs sources are together.